index.js

```diff
@@ -186,6 +188,12 @@ HomeAssistantPlatform.prototype = {
           accessory = new HomeAssistantSwitch(that.log, entity, that, 'input_boolean')
         }else if (entity_type == 'fan'){
           accessory = new HomeAssistantFan(that.log, entity, that)
+        }else if (entity_type == 'binary_sensor'){
+          if (entity.attributes && entity.attributes.sensor_class == 'opening') {
+            accessory = new HomeAssistantBinarySensor(that.log, entity, that, Service.Door, Characteristic.CurrentPosition, 100, 0)
```

Comment:
Looks like for `Service.Door` these characteristics are required:

- `Characteristic.CurrentPosition`
- `Characteristic.PositionState`
- `Characteristic.TargetPosition`